### ISSUE TITLE
[enchantment] Select first day of week

### ISSUE THREAD

**Issue by o-murphy** [noXplode/aiogram_calendar#18]
In some locales the first day of a week is a Saturday, we can to add support of this feature

**Comment by noXplode** [comment]
As far as I can tell, locale or calendar modules does not have a feature of automatically setting first day of week in calendar, we can just set it through `calendar.setfirstweekday(6)` for example. Do you think theres a point of adding parameter of setting first day of week?

**Comment by o-murphy** [comment]
> As far as I can tell, locale or calendar modules does not have a feature of automatically setting first day of week in calendar from locale value, we can just set it manually through `calendar.setfirstweekday(6)` for example. Do you think theres a point of adding parameter of setting first day of week?

Yes you're right, locale or calendar don't have this feature. In common people uses 2 different variants is Monday or Sunday, so we need no implement all of them. Monday is compatible with ISO8601 standard, but Sunday uses in a Greco-Roman style (in USA, Japan etc) We can add such an optional flag kinda is_sunday_first to the __init__ method, to not create a separate method for this feature

**Comment by o-murphy** [comment]
Now I think about the refactoring of month keyboard generator, cause it can be simpler I think so. And we need to check if python's calendar is accept Sunday as first weekday. This feature is not as important, so we don't need to take it from User props.

**Comment by o-murphy** [comment]
So, we can use calendar.setfirstweekday if the __init__ accept an additional argument, what's have to be 1 by default
